pkgs/development/python-modules/urllib3/default.nix

```diff
@@ -65,7 +64,7 @@ buildPythonPackage rec {
 
   passthru.optional-dependencies = {
     brotli = if isPyPy then [ brotlicffi ] else [ brotli ];
-    secure = [ certifi cryptography idna pyopenssl ];
+    secure = [ certifi cryptography idna ];
```

Comment:
Do we have any we can link in urllib3 that mentions that pyopenssl is not required? If so we should link that here that pyopenssl is not added back later.

Reply:
How about https://urllib3.readthedocs.io/en/latest/reference/contrib/pyopenssl.html